### DISCUSSION TITLE
High Score Bug after finishing Campaign

### DISCUSSION THREAD

**Discussion by Horrorla** [ihhub/fheroes2#5455] [General]
Hey, I am testing the remake of homm2 and I really enjoy it. After I finished the Evil Campaign there was no entry in the high score list. In general I have a black screen when the dragon is breathing at the board and just see my standing afterwards.
For normal maps the High Scores works, but in my version it doesn´t works at the Campaign. When I started the game and open the high score window I just can see the normal map score and the Campaign
 one is gray (see picture below)
Does anyone else has the same issue like me ?

<img width="512" alt="MissingCampaignHighscore" src="https://user-images.githubusercontent.com/106203359/170147839-1bc4bc2c-39df-4ebc-99db-fdedeffd6a98.png">

**Comment by oleg-derevenetz** [comment]
Hi @Horrorla high scores support for campaigns was added just yesterday, so you need to install the latest snapshot to get it to work.

**Reply by Horrorla** [reply]
@oleg-derevenetz  thank you for this fast answer. I will upgrade to the latest version and continue testing the game!

**Reply by oleg-derevenetz** [reply]
@Horrorla not the latest version, but the latest developer snapshot from [here](https://ihhub.github.io/fheroes2/INSTALL.html#snapshots-latest-builds). These changes are not yet available in the latest version.
